docs/docs/03-syntax-and-usage/10-template-composition.md

```diff
@@ -235,6 +235,49 @@ func main() {
 	<p>Dynamic contents</p>
 </div>
 ```
+## Joining Components
+With your templ components you can use `templ.Join` to generate a single `templ.Component` from your components.
+```templ
```

Comment:
Tabs not spaces ;)

Also I think that joining generated templ components here would be more intuitive for the docs:

templates.go
```templ
package main

templ hello() {
	<span>hello</span>
}

templ world() {
	<span>world</span>
}

templ helloWorld() {
	@templ.Join(hello(), world())
}
```

main.go
```go
package main

import (
	"context"
	"os"
)

func main() {
	helloWorld().Render(context.Background(), os.Stdout)
}
```

Output:
```html
<span>hello</span><span>world</span>
```